src/main/java/xyz/vegaone/easytracking/config/EasyTrackConfig.java

```diff
@@ -23,5 +23,4 @@ public DataSource dataSource() {
 
```

Comment:
For some reason I cannot comment on line 14th, but you have a field injection there. Could you please switch to constructor based injection?